site/en/install/_index.yaml

```diff
@@ -26,7 +26,10 @@ landing_page:
             </ul>
           </td><td>
             <ul>
-              <li>macOS 10.12.6 (Sierra) or later (no GPU support)</li>
+              <li>
+                macOS &gt;=10.12.6 (Sierra) but &lt;11 (Big Sur) (no GPU support).
+                For Big Sur and GPU support, use <a href="https://github.com/apple/tensorflow_macos">tensorflow_macos</a>.
```

Comment:
Isn't that one only for TF on the M1 chip?

Reply:
Readme says "hardware acceleration is supported on Macs with M1 and Intel-based Macs through Apple’s ML Compute framework."
Though the TF version lags behind a bit.

Reply:
I'm on an x86 Mac running macOS 11. tensorflow_macos installs and runs OK for me (so far). Mainline Tensorflow does not.

Reply:
Hi @jameshfisher, macOS 11.1 here without M1  (11.3 beta is the latest, I think). Installing TF2 with PiPy worked as recently as January.

Reply:
@8bitmp3 interesting - how exactly did you install it? The standard `pip install tensorflow` worked for you?

Reply:
@jameshfisher Yep. Have you tried setting up a new tf2 virtualenv/docker for a "clean-install"?

Reply:
@8bitmp3 interesting - I tried it with virtualenv, and it installs! Thanks. However, it's very unclear why virtualenv makes a difference, and the TensorFlow install instructions don't say to use virtualenv.

So the mystery deepens on https://github.com/tensorflow/tensorflow/issues/47205. 

Docker I would consider cheating - it's basically installing on Linux, not macOS.

@8bitmp3 which method are you using on macOS? virtualenv or docker?

Reply:
I'm glad this helped @jameshfisher. I use `virtualenv` or `pyenv` because on macOS you can end up with too many Python interpreters. Check out this post: https://opensource.com/article/19/5/python-3-default-mac (note that homebrew tends to installs the latest one even if you don't need it).

![](https://opensource.com/sites/default/files/uploads/python_environment_xkcd.png)